vcs/git/clone_update.go

```diff
@@ -93,10 +93,11 @@ func (r *Repository) UpdateEverything(opt vcs.RemoteOpts) error {
 	}
 
 	if err := rm.Fetch([]string{"+refs/*:refs/*"}, &opts, ""); err != nil {
-		return err
+		return vcs.UpdateResult{}, err
 	}
 
-	return nil
+	// TODO: Calculate value of vcs.UpdateResult.
```

Comment:
How come? I've updated the [README's Implementation Differences](https://github.com/sourcegraph/go-vcs#implementation-differences) table. Do you think that's not enough?